web/extensions/core/keybinds.js

```diff
@@ -0,0 +1,74 @@
+import { app } from "/scripts/app.js";
+import { $el } from "/scripts/ui.js";
+
+const id = "Comfy.Keybinds";
+app.registerExtension({
+	name: id,
+	init() {
+		const keybindListener = function(event) {
+			const target = event.composedPath()[0];
+
+			if (target.tagName === "INPUT" || target.tagName === "TEXTAREA") {
+				return;
+			}
+
+			const modifierPressed = event.ctrlKey || event.metaKey;
+
+			// Queue prompt using ctrl or command + enter
+			if (modifierPressed && (event.key === "Enter" || event.keyCode === 13 || event.keyCode === 10)) {
```

Comment:
`keyCode` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) (and `key` is `Enter` for both enter buttons)

Reply:
Yeah it is, but I believe comfy does want to have keycode atleast as a backup as they created this PR (https://github.com/jagenjo/litegraph.js/pull/378) using keycodes because of their remapped keyboard. Also just that function previously existed (from #79) so I based the rest off of that.

Also idk wym by both enter buttons. From [docs here](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values) it seems to be only one Enter button, or the platforms with multiple they all seem logical to also work like the keypad enter.

Reply:
> comfy does want to have keycode atleast as a backup

Fair enough

> Also idk wym by both enter buttons.

![Enter](https://user-images.githubusercontent.com/4073789/232183089-96bdf313-466b-4d79-838a-33fd440a7d39.png)

Here's a good tool for keyboard events.
https://www.toptal.com/developers/keycode

Reply:
The link you sent shows that `event.code` is different for Numpad and the normal enter button, but we aren't using `event.code` at all, as that does not work with remapped boards as it always gives us the physical key location. 

I did mention the keypad enter in my comment before, but I still no see issue using both enter buttons, as they are both enters, it still makes sense to me and probably did to pythongoess when he wrote it originally.

Reply:
I didn't say `code` is the same, `key` is the same.

Anyway, I think I just misread it. I saw this
```python
event.keyCode === 13 || event.keyCode === 10
```
and assumed you were checking for both enter keys, but then I noticed they're both 13. What's 10?

Reply:
10 seems to be the line carriage character, but info about it is very sparse and it seems to rarely occur. I don't know if it is necessary, it is pythongoess who added that part. I think we should just leave it in, as this is all for backwards compatibility anyway and handling as many cases that present itself to be similar/the same is better than not.